cloudinit/net/__init__.py

```diff
@@ -345,6 +345,8 @@ def find_fallback_nic_on_freebsd(blacklist_drivers=None):
     we'll use the first interface from ``ifconfig -l -u ether``
     """
     stdout, _stderr = util.subp(['ifconfig', '-l', '-u', 'ether'])
+    LOG.debug('ifconfig -l -u ether output: (%s, %s)', stdout, _stderr)
```

Comment:
if you put the command in a variable, you don't have to repeat yourself here

Reply:
Yes, it's not really elegant. I plan to remove the commit when the problem will be resolved.